src/microbe_stage/editor/tooltips/SelectionMenuToolTip.tscn

```diff
@@ -138,14 +140,36 @@ margin_right = 333.0
 margin_bottom = 132.0
 mouse_filter = 2
 
-[node name="HSeparator" type="HSeparator" parent="MarginContainer/VBoxContainer"]
+[node name="CanBeModified" type="HBoxContainer" parent="MarginContainer/VBoxContainer"]
 margin_top = 147.0
 margin_right = 333.0
-margin_bottom = 151.0
+margin_bottom = 166.0
+
+[node name="UpgradeIcon" type="TextureRect" parent="MarginContainer/VBoxContainer/CanBeModified"]
+modulate = Color( 1, 0.898039, 0.337255, 1 )
+margin_right = 16.0
+margin_bottom = 19.0
+rect_min_size = Vector2( 16, 16 )
+texture = ExtResource( 2 )
+expand = true
+stretch_mode = 6
+
+[node name="CanBeModifiedLabel" type="Label" parent="MarginContainer/VBoxContainer/CanBeModified"]
+margin_left = 20.0
+margin_right = 192.0
+margin_bottom = 19.0
+custom_colors/font_color = Color( 1, 0.898039, 0.337255, 1 )
+custom_fonts/font = ExtResource( 10 )
+text = "MODIFY_ONCE_PLACED"
```

Comment:
I think the text key would be better to have something like "PART_IS_MODIFIABLE"